### PR TITLE
respect color preview pref in console

### PR DESCRIPTION
**Pull request by kevinushey**
### Intent

Addresses https://github.com/rstudio/rstudio/issues/13118.

### Approach

We normally try to respect preference changes for editor instances here:

https://github.com/rstudio/rstudio/blob/main/src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/TextEditingTargetPrefsHelper.java#L128-L132

However, this only applies to Ace instances in the Source pane; not in the Console pane. This PR makes sure we apply the preference for Ace instances in the Console as well.

### Automated Tests

N/A

### QA Notes

N/A

### Documentation

N/A

### Checklist

- [x] If this PR adds a new feature, or fixes a bug in a previously released version, it includes an entry in `NEWS.md` 
- [x] If this PR adds or changes UI, the updated UI meets [accessibility standards](https://github.com/rstudio/rstudio/wiki/Accessibility)
- [x] A reviewer is assigned to this PR (if unsure who to assign, check Area Owners list)
- [x] This PR passes all local unit tests

### CODE REVIEW

File: src/node/desktop/envvars.bat [hunk 1]
```diff
@@ -4,6 +4,7 @@ REM Run this script with 'call envvars.bat' to update the PATH in your
 REM cmd.exe shell, so that the appropriate versions of node are found
 set NODE_PATHS=^
 	..\..\..\dependencies\common\node\16.20.2^
+	..\..\dependencies\common\node\16.20.2^
```
Comment (jgutman) [inline src/node/desktop/envvars.bat:7]: Why is this here? seems like the dependencies folder 3 levels up is correct

**Review by jgutman — APPROVED** [review]

File: src/node/desktop/envvars.bat (hunk 1 above)
Comment (kevinushey) [reply src/node/desktop/envvars.bat:7]: Because I needed to source this from the `src/gwt` folder, because the build tasks were failing to find node. So really ... just a way to be lazy.
